src/App/Pages/Accounts/EnvironmentPageViewModel.cs

```diff
@@ -57,5 +72,16 @@ public async Task SubmitAsync()
 
             SubmitSuccessAction?.Invoke();
         }
+
+        public bool FormValidation()
+        {
+            BaseUrlValid = string.IsNullOrEmpty(BaseUrl) || Uri.IsWellFormedUriString(BaseUrl, UriKind.Relative);
+            ApiUrlValid = string.IsNullOrEmpty(ApiUrl) || Uri.IsWellFormedUriString(BaseUrl, UriKind.Relative);
+            IdentityUrlValid = string.IsNullOrEmpty(IdentityUrl) || Uri.IsWellFormedUriString(BaseUrl, UriKind.Relative);
+            WebVaultUrlValid = string.IsNullOrEmpty(WebVaultUrl) || Uri.IsWellFormedUriString(BaseUrl, UriKind.Relative);
+            IconsUrlValid = string.IsNullOrEmpty(IconsUrl) || Uri.IsWellFormedUriString(BaseUrl, UriKind.Relative);
+
+            return BaseUrlValid && ApiUrlValid && IdentityUrlValid && WebVaultUrlValid && IconsUrlValid;
+        }
```

Comment:
A couple of things:
- Unless you're doing or plan to do something with these public properties, e.g. `BaseUrlValid`, I'd just use local variables inside this method. Right now they are being "cached" with no use.
- Also, you're using `Uri.IsWellFormedUriString(BaseUrl, UriKind.Relative )` on all cases, it should change to each url case, i.e. `ApiUrlValid`, `IdentityUrlValid`, and so on
- Also, I'd change the  name of the method to something more readable => `public bool ValidateUrls()` or `public bool AreUrlsValid()`
- Furtheremore, you could have a local function and just return using the && concatenation of the call to it with each url, i.e.

```c#
public bool ValidateUrls()
{
    bool IsUrlValid(string url)
    {
        return string.IsNullOrEmpty(url) || Uri.IsWellFormedUriString(url, UriKind.Relative);
    }

    return IsUrlValid(BaseUrl)
           &&
           IsUrlValid(ApiUrl)
           &&
           IsUrlValid(IdentityUrl)
           &&
           IsUrlValid(WebVaultUrl)
           &&
           IsUrlValid(IconsUrl);
}
```